### PR TITLE
Documented 3 more default plugins

### PR DESCRIPTION
**Pull request by JoanneOFlynn2018**
disk, hugepages, interface

### CODE REVIEW

File: common/collectd/ref_collectd-plugins.adoc [hunk 1]
```diff
@@ -542,10 +637,32 @@ Use the `disk` plugin to collect performance statistics of hard-disks and, if su
 
 [discrete]
 == collectd::plugin::interface
-* collectd::plugin::interface::interfaces
-* collectd::plugin::interface::ignoreselected
-* collectd::plugin::interface::reportinactive
-* Collectd::plugin::interface::interval
+
+The `interface` plugin is enabled by default. Use this plugin to measure interface traffic in octets, packets per second, and error rate per second.  of interfaces.
```
Comment (JoanneOFlynn2018) [inline common/collectd/ref_collectd-plugins.adoc:641]:
```suggestion
Use the `interface` plugin to measure interface traffic in octets, packets per second, and error rate per second. This plugin is enabled by default. 
```

**Review by mrunge — APPROVED** [review]
Thank you Joanne. This looks good to me.

**Review by leifmadsen — APPROVED** [review]

File: common/collectd/ref_collectd-plugins.adoc [hunk 2]
```diff
@@ -471,12 +504,54 @@ For more information about configuring the `df` plugin, see https://collectd.org
...
+|===
+
+
+.disk metrics
+[cols="20%,30%,50%"]
+|===
+|Name  |Description
+
+| merged
+| The number of operations that can be merged together, already queued operations, for example, one physical disk access served two or more logical operations.
```
Comment (leifmadsen) [inline common/collectd/ref_collectd-plugins.adoc:527]: @mrunge the only question @JoanneOFlynn2018 and I had this morning was around this sentence. Can this be refined or some additional context be added?

Comment (mrunge) [reply common/collectd/ref_collectd-plugins.adoc:527]: Some disks will re-order commands sent to the disk and also will merge write requests together. This gives an insight on operations specific for the disk.
I am not sure how to describe this better, or if this is the place to describe disk-internal optimizations.

Comment (leifmadsen) [reply common/collectd/ref_collectd-plugins.adoc:527]: Yea I don't think it is. We should just say less and let others determine how disk operations work.

**Comment by leifmadsen** [comment]
@JoanneOFlynn2018 based on the feedback from @mrunge I think we can just merge this.

**Pull request merged**